arm/Microsoft.Network/virtualNetworks/deploy.bicep

```diff
@@ -37,8 +37,8 @@ param diagnosticEventHubAuthorizationRuleId string = ''
 param diagnosticEventHubName string = ''
 
 @allowed([
-  'CanNotDelete'
   'NotSpecified'
+  'CanNotDelete'
```

Comment:
Same as Microsoft.AnalysisServices/servers. Probably can be reverted back?

Reply:
Fixed